### ISSUE TITLE
Missing to serve the request if multiple requests are on Queue

### ISSUE THREAD

**Issue by biku-gdrive** [google/volley#298]
On my application we are syncing lots of request and all are in different priorities and some are same. Observed Volley lib fail to serve the response back. even we are not getting timeout. 
around 50 requests are on queue out of that 2 are failing to receive the response back. 

Did any one face this issue and whats the fix for this ?

**Comment by jpd236** [comment]
I'm not aware of anything inherent to Volley that would cause this. The queues have no inherent capacity, and any uncanceled request should always dispatch either a listener or an error listener unless the underlying HTTP stack hangs forever.

There's no action here that I can see for us to take here to investigate, at least without more info. The best thing would be if you can provide a small sample app that reproduces the problem. You could also consider turning on Volley's detailed request logging (adb shell setprop log.tag.Volley VERBOSE, then restart your app) and viewing the logs to see which requests are being dropped and where they are dropped (and/or upload them here for inspection). You might also consider dumping the stack traces for all threads in your application to see where they are hanging.

Please reopen if you can provide any more details.

**Comment by biku-gdrive** [comment]
Thanks Jeff,

For your quick response, I observed this issue when we have more request
with Normal priority, Our application sync lots of data including Media
files [Video/audio/image] and also huge text data. out of 50 requests 2
response are dropped, it never receive any response. The problem is on my
test app i executed 1000 request it works fine but on our app we receive
dynamic resource urls which are keep adding to the request and there it
fail to get the response. These issues was reported by my client side
testers and i am not able to trace what was the issue, neither i am able to
reproduce that issue.

 which i can't use it outside. Now i just hacked it by changing the
priorities to High and we started receiving the response back but my worry
is even if we have 1000 request of same request application should not miss
any response but Volley fail to serve them. I am monitoring this issue if i
face the same issue i will try to collect the log and share it with you.

Thanks
Bikash
